go-apps/meep-gis-engine/api/swagger.yaml

```diff
@@ -283,6 +350,50 @@ definitions:
     example:
       active: true
       type: "MOBILITY"
+  DistanceParameters:
+    type: "object"
+    properties:
+      assetName:
+        type: "string"
+        description: "Asset name of a second element for query purpose. If present,\
+          \ latitude and longitude are ignored."
+      latitude:
+        type: "number"
+        format: "float"
+        description: "Latitude of a second element for query purpose. Taken into account\
+          \ only if AssetName is not present."
+      longitude:
+        type: "number"
+        format: "float"
+        description: "Longitude of a second element for query purpose. Taken into\
+          \ account only if AssetName is not present."
+    description: "Parameters for distance query purpose."
+    example:
+      latitude: 0.8008282
+      assetName: "assetName"
+      longitude: 6.0274563
+  DistanceResponse:
+    type: "object"
+    required:
+    - "distance"
+    properties:
+      distance:
+        type: "number"
+        format: "float"
+        description: "Distance between two points (in meters)"
+      latitude:
+        type: "number"
+        format: "float"
+        description: "Destination asset latitude"
+      longitude:
+        type: "number"
+        format: "float"
+        description: "Destination asset longitude"
```

Comment:
Why return these? Are they the coordinates that were passed in the request?
Should we return the src & dest coordinates if we want to include these values?

Reply:
is meant to be an optional parameter so that the api can return it or not for the destination only, not the source point. Reason behind it is because the source point is a parameter while the response is just related to the destination that was in the body. It is assumed only the destination information is requested since the source should know where it is located.

Reply:
Okay, it makes sense if the target point is an asset name that we may want to return its coordinates as well. If the Target point is a lat/long I assume you don;t return the lat/long, correct?